src/plugins/intel_gpu/src/graph/program_node.cpp

```diff
@@ -1636,19 +1655,19 @@ void program_node::init_onednn_primitive_attributes() {
                 // 1. clamp
                 {
                     if (q_param->_need_clamp) {
-                        auto in_lo = get_dependency(dep_idx++).get_output_layout();
-                        auto in_hi = get_dependency(dep_idx++).get_output_layout();
+                        auto in_lo = get_input_layout(dep_idx++);
+                        auto in_hi = get_input_layout(dep_idx++);
                         dnnl::algorithm clamp_max = dnnl::algorithm::binary_max;
                         dnnl::algorithm clamp_min = dnnl::algorithm::binary_min;
                         dnnl::memory::desc in_lo_desc = onednn::layout_to_memory_desc(in_lo, dnnl::memory::format_tag::ab, true);
                         dnnl::memory::desc in_hi_desc = onednn::layout_to_memory_desc(in_hi, dnnl::memory::format_tag::ab, true);
 
                         post_ops.append_binary(clamp_max, in_lo_desc);
                         update_onednn_post_op_list(onednn_post_op_type::binary_max, dep_idx - 2, dnnl::memory::format_tag::ab, true,
-                                                   in_lo_desc.get_dims(), in_lo_desc.get_data_type());
+                                                    in_lo_desc.get_dims(), in_lo_desc.get_data_type());
                         post_ops.append_binary(clamp_min, in_hi_desc);
                         update_onednn_post_op_list(onednn_post_op_type::binary_min, dep_idx - 1, dnnl::memory::format_tag::ab, true,
-                                                   in_hi_desc.get_dims(), in_hi_desc.get_data_type());
+                                                    in_hi_desc.get_dims(), in_hi_desc.get_data_type());
```

Comment:
are you intended to change such kind of formatting? I don't think it is a good idea to enforce such formatting from your local dev environment..

Reply:
I didn't do it on purpose, I think the code was modified during the copying process to create new method.
rollback it.